cookbook/upgrade/_update_all_packages.rst.inc

```diff
@@ -0,0 +1,15 @@
+You may also want to upgrade the rest of your libraries. If you've done a
+good job with your `version constraints`_ in ``composer.json``, you can do
+this safely by running:
+
+.. code-block:: bash
+
+    $ composer update
+
+.. caution::
+
+    Beware, if you have some bad
+    `version constraints <https://getcomposer.org/doc/01-basic-usage.md#package-versions>`_
```

Comment:
The referenced URL should be removed to the bottom of the file (this should work in included files too, doesn't it?).